backend/src/validation/auth.ts

```diff
@@ -0,0 +1,134 @@
+import { z } from "zod";
+
+export const BeginEmailSignUpV1 = z.object({
+  body: z.object({
+    email: z.string().email().trim()
+  })
+});
+
+export const VerifyEmailSignUpV1 = z.object({
+  body: z.object({
+    email: z.string().email().trim(),
+    code: z.string().email().trim()
```

Comment:
This shoud be just a string not a email

Reply:
Can you also go through other aspects of app like inviting users, downloading PDF, and all other actions you can think of. The PR is too large to so we may miss these cases. I'm going through some testing but it would be good if you can also go through this too